src/server/services/focus/focus-toxx-helpers.ts

```diff
@@ -125,6 +125,7 @@ export function createToxxItemStep(
     title: labelSet.stepType,
     description: getDocumentStepDescription(document, labelSet.labels),
     datePublished,
+    decision: labelSet.decision,
```

Comment:
Misschien kun je dit beter in een condition zetten. Zodoende komt de decision prop niet op de stappen te staan die geen decision hebben.

```
if (labelSet.stepType === 'besluit' && labelSet.decision) { item.decision = labelSet.decision };
```